cves/kernel/CVE-2017-17805.yml

```diff
@@ -106,9 +118,9 @@ vcc_instructions: |
   Place any notes you would like to make in the notes field.
 vccs:
 - commit: 974e4b752ee623854c5dc2bbfc7c7725029ce173
-  note: Discovered automatically by archeogit.
+  note: Discovered automatically by archeogit. Salsa20 initially introduced.
 - commit: eb6f13eb9f812f5812ed5d14f241309da369dee6
-  note: Discovered automatically by archeogit.
+  note: Discovered automatically by archeogit. Main bug introduced.
 - commit: 2407d60872dd2a95404c6048f775f3b64d438f4b
   note: Discovered automatically by archeogit.
 upvotes_instructions: |
```

Comment:
I give this vulnerability 6 upvotes.

Reply:
Done thank you!

Reply:
Love how you included what the commits were responsible for !

Reply:
Thank you!